utils/build-script-impl

```diff
@@ -3555,8 +3555,6 @@ for host in "${ALL_HOSTS[@]}"; do
                             continue
                         ;;
                     esac
-                else
-                    INSTALL_TARGETS="install-lldb"
```

Comment:
`build-script` is fairly complex so I'm probably missing something here, but ... your review just removes an install target? Where do we actually perform the installation for CMake?

Reply:
IIUC `INSTALL_TARGETS` collects a list of targets that will be installed (don't ask me where). It's is initialized with `install` here: https://github.com/apple/swift/blob/2fa436e53e/utils/build-script-impl#L3491 This should be fine for LLDB.

Reply:
We could go for `install-distribution` with https://github.com/apple/swift-lldb/pull/1785